packages/lib/profile/service.ts

```diff
@@ -145,34 +134,42 @@ export const updateProfile = async (
 
 const deleteUser = async (userId: string): Promise<TProfile> => {
   validateInputs([userId, ZId]);
+
   const profile = await prisma.user.delete({
     where: {
       id: userId,
     },
     select: responseSelection,
   });
-  revalidateTag(getProfileByEmailCacheTag(profile.email));
-  revalidateTag(getProfileCacheTag(userId));
+
+  profileCache.revalidate({
```

Comment:
we also need to revalidate by id

Reply:
in this code, there is a bit of a confusion, `id === userId`. The userId passed as an argument is what is `profile.id`.

So what I am doing is to remove the `byId` and replace it with `byUserId`.

What do you think? or should I go the other way around?